README.md

```diff
@@ -2,4 +2,4 @@
 
 [![Build Status](https://travis-ci.org/gregcaporaso/mockrobiota.svg?branch=master)](https://travis-ci.org/gregcaporaso/mockrobiota)
```

Comment:
I guess the README change could have been to update ``gregcaporaso`` to ``caporaso-lab`` in these links. We can do that here or in another.